packages/lesmis-server/src/routes/PDFExportRoute.ts

```diff
@@ -38,12 +38,17 @@ export class PDFExportRoute extends Route<PDFExportRequest> {
   };
 
   private verifyBody = (body: any): Body => {
+    const legitLESMISHostnames = ['lesmis.la', 'www.lesmis.la'];
```

Comment:
I'd suggest having an explicit list of the domains that Tupaia supports

```suggestion
    const validDomains= ['.tupaia.org', '.lesmis.la'];
```

And then later do:
```
if (!validDomains.some(domain=> location.hostname.endsWith(domain)) && !location.hostname.endsWith('localhost')) {
```

Might even be best to store this list of valid hostnames in a utils file/function somewhere, but I can't see another area in the code where do we such a check, so it seems okay to me for it to be here for now 👍

Reply:
Thanks for the renaming suggestion. 
This check is for security, because `pdf-export-server` will access provided `pdfPageUrl`, and without this check, hacker could easily get user's session by a malicious link as `pdfPageUrl`.

Reply:
Just found that we have to keep `lesmis.la` because it is a legit DNS for lesmis as well.